app/bundles/FormBundle/Model/FormModel.php

```diff
@@ -423,6 +423,11 @@ public function generateHtml(Form $entity, $persist = true): string
         if ($this->twig->getLoader()->exists('@themes/'.$theme.'/html/MauticFormBundle/Builder/_style.html.twig')) {
             $styleToRender = '@themes/'.$theme.'/html/MauticFormBundle/Builder/_style.html.twig';
         }
+
+        if ($this->twig->getLoader()->exists('@themes/'.$theme.'/html/MauticFormBundle/Builder/style.html.twig')) {
+            $styleToRender = '@themes/'.$theme.'/html/MauticFormBundle/Builder/style.html.twig';
+        }
```

Comment:
I know it's silly in this case since we are talking about 3 lines, but there is a lot of duplication. Basically just 1 character difference. Can we do something like this?
```suggestion
        foreach (['_style', 'style'] as $styleFile) {
            $stylePath = "@themes/{$theme}/html/MauticFormBundle/Builder/{$styleFile}.html.twig";
            if ($this->twig->getLoader()>exists($stylePath)) {
                $styleToRender = $stylePath;
            }
        }
```
Or use a private method.